test/indexer/test_hca_indexer.py

```diff
@@ -651,10 +652,22 @@ def test_cell_line_sample(self):
                 sample_entity_type = sample['entity_type']
                 if aggregate:
                     document_ids = one(contents[sample_entity_type])['document_id']
+                    self.assertEqual(one(contents['specimens'])['organ'], ['blood'])
+                    self.assertEqual(one(contents['specimens'])['organ_part'], ['venous blood'])
+                    self.assertEqual(one(contents['cell_lines'])['model_organ'], ['blood (cell_line)'])
+                    self.assertEqual(one(contents['cell_suspensions'])['organ'], ['blood (cell_line)'])
+                    self.assertEqual(one(contents['cell_suspensions'])['organ_part'], [None])
                 else:
                     document_ids = [d['document_id'] for d in contents[sample_entity_type]]
                     entity = one([d for d in contents[sample_entity_type] if d['document_id'] == sample['document_id']])
                     self.assertEqual('cell_line_Day7_hiPSC-CM_BioRep2', entity['biomaterial_id'])
+                    self.assertEqual(one(contents['specimens'])['organ'], 'blood')
```

Comment:
Almost the same as the line in the if branch. Only difference is that RHS of comparison is `["…"]` vs `"…"`

Reply:
Refactored test case to minimize duplication